Doc/library/time.rst

```diff
@@ -393,12 +395,21 @@ Functions
    <https://learn.microsoft.com/windows-hardware/drivers/kernel/high-resolution-timers>`_
    which provides resolution of 100 nanoseconds. If *secs* is zero, ``Sleep(0)`` is used.
 
-   Unix implementation:
+   .. rubric:: Unix implementation
 
    * Use ``clock_nanosleep()`` if available (resolution: 1 nanosecond);
    * Or use ``nanosleep()`` if available (resolution: 1 nanosecond);
    * Or use ``select()`` (resolution: 1 microsecond).
 
+   .. note::
+
+      For polling, consider using :meth:`select.poll.poll(0) <select.poll.poll>`
+      instead of ``time.sleep(0)``. To emulate a "no-op", use :keyword:`pass`.
```

Comment:
I don't think that it's the right place to mention polling file descriptors in time.sleep() documentation.